### PR TITLE
Create works and about me sections

### PR DESCRIPTION
**Pull request by ebidien**
The following changes were introduced in the `create-works-and-about-me-sections` feature branch:
- Create a works section with project cards
- Create an about me section with an intro, tech stack, and skills
- Fix linter errors for duplicate properties

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-1.png" alt="project-snapshot">
+            <h2 class="project-title">Tonic</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
+            </div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like font, color, spacing, etc. See the color of `Back End Dev and 2015`. See image below for reference
### Your design

your

### Figma design
fimga

File: index.html [hunk 2]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-2.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like font, color, spacing, etc. See the color of `Back End Dev and 2015`. See image below for reference
### Your design

your

### Figma design
fimga

File: index.html [hunk 3]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-3.png" alt="project-snapshot">
+            <h2 class="project-title">Tonic</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like font, color, spacing, etc. See the color of `Back End Dev and 2015`. See image below for reference
### Your design

your

### Figma design
fimga

File: index.html [hunk 4]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+        </div>
+        <div class="card-works">
+            <img class="snapshot" src="work-snapshot-4.png" alt="project-snapshot">
+            <h2 class="project-title">Multi-Post Stories</h2>
+            <div class="caption-container">
+                <div class="text">CANOPY</div>
+                <div class="dot">&middot;</div>
+                <div class="text">Back End Dev</div>
+                <div class="dot">&middot;</div>
+                <div class="text">2015</div>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] You should stick to the design as much as possible like font, color, spacing, etc. See the color of `Back End Dev and 2015`. See image below for reference
### Your design

your

### Figma design
fimga

File: index.html [hunk 5]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+            <button class="project-link" type="button">See Project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-container">
+            <h1 class="about-title">About Myself</h1>
+            <div class="about-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
```
Comment (roshan-bajgain) [inline index.html:135]: - [ ] You should stick to the design as much as possible. pleas adjust the spacing, see image below for reference
### Your design
your


### Figma design
fimga

File: index.html [hunk 6]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+                            alt="github" /></a></li>
+                <li class="social-media-icon"><a href="https://angel.co/u/dienebi-ombu"><img src="angellist.png"
+                            alt="angellist" /></a></li>
+            </ul>
+            <button class="resume-link" type="button">Get my resume</button>
+            <section class="about-stack">
+                <div class="stack-container">
+                    <div class="stack-category">
+                        <h2>Languages</h2>
+                        <i class="fa fa-chevron-down"></i>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Please use this color `#ACB7C3` for the drop-down menu to match the design

File: index.html [hunk 7]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+                    </div>
+                    <ul class="category-list">
+                        <li class="category-item"><img class="stack-icon" src="js-icon.png" alt=""><h3 class="stack-item">JavaScript</h3></li>
+                        <li class="category-item"><img class="stack-icon" src="html-icon.png" alt=""><h3 class="stack-item">HTML</h3></li>
+                        <li class="category-item"><img class="stack-icon" src="css-icoon.png" alt=""><h3 class="stack-item">CSS</h3></li>
+                    </ul>
+                </div>
+                <div class="stack-category">
+                    <h2>Frameworks</h2>
+                    <i class="fa fa-chevron-right"></i>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Please use this color `#ACB7C3` for the drop-down menu to match the design

File: index.html [hunk 8]
```diff
@@ -42,5 +44,134 @@ <h2>LET'S CONNECT</h2>
...
+                </div>
+                <div class="stack-separator"></div>
+                <div class="stack-category">
+                    <h2>Skills</h2>
+                    <i class="fa fa-chevron-right"></i>
```
Comment (roshan-bajgain) [inline index.html]: - [ ] Please use this color `#ACB7C3` for the drop-down menu to match the design

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @ebidien, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 8 above)
Comment (ebidien) [reply index.html]: Corrected the color inline thus: `<i class="fa fa-chevron-right" style="color: acb7c3;"></i>`

File: index.html (hunk 7 above)
Comment (ebidien) [reply index.html]: Corrected the color inline thus: `<i class="fa fa-chevron-right" style="color: acb7c3;"></i>`

File: index.html (hunk 6 above)
Comment (ebidien) [reply index.html]: Corrected the color inline thus: `<i class="fa fa-chevron-down" style="color: acb7c3;"></i>`

File: index.html (hunk 5 above)
Comment (ebidien) [reply index.html:135]: about-self

File: index.html (hunk 4 above)
Comment (ebidien) [reply index.html]: canopy

File: index.html (hunk 3 above)
Comment (ebidien) [reply index.html]: canopy

File: index.html (hunk 2 above)
Comment (ebidien) [reply index.html]: canopy

File: index.html (hunk 1 above)
Comment (ebidien) [reply index.html]: canopy

**Comment by ebidien** [comment]
@roshan-bajgain, the requested changes have been made.

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @ebidien  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Linters are passing ✔️
## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**